### PR TITLE
Feature/dsi 5706 audit transporter

### PR DESCRIPTION
**Pull request by Avinash-teky**

### CODE REVIEW

File: src/app/user/adapter/UserSequelizeAdapter.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const generateSalt = require('./../utils/generateSalt');
 const { v4: uuid } = require('uuid');
 const { promisify } = require('util');
 const crypto = require('crypto');
+const config = require('./../../../infrastructure/config');
```
Comment (ipredoi) [inline src/app/user/adapter/UserSequelizeAdapter.js:12]: Can we remove line 12 - `config` is never used

Comment (Avinash-teky) [reply src/app/user/adapter/UserSequelizeAdapter.js]: Removed unused import @ipredoi

**Review by ipredoi — CHANGES_REQUESTED** [review]
Can we update the unit tests broken by this change?

**Comment by Avinash-teky** [comment]
@ipredoi Added issue https://dfe-secureaccess.atlassian.net.mcas.ms/browse/DSI-5889 to resolve failing test cases

**Review by ipredoi — APPROVED** [review]
Unit tests to be fixed as part of https://dfe-secureaccess.atlassian.net.mcas.ms/browse/DSI-5889

**Pull request merged**